### ISSUE TITLE
Adding SeqSeg to SimVascular

### ISSUE THREAD

**Issue by numisveinsson** [SimVascular/SimVascular#1070]
**SeqSeg could make model construction significantly faster.**

SeqSeg is written in **python**:
[Click here for repo](https://github.com/numisveinsson/SeqSeg)

Main dependencies:
- pytorch
- simpleitk
- vtk
- numpy
- nnunet

For running, SeqSeg requires:

1. **Model weights** (pretrained) to be downloaded
2. User input of **two seed points and radius estimate** of blood vessel of interest
3. Optional hyperparameters: a) **number of steps** to take, b) **number of branches** to add

Idea for implementation:
Add as plugin to GUI at top row.
Build of off 3D levelset code (which also requires seed points)

**Comment by ktbolt** [comment]
I don't think this is going to happen.
